### PR TITLE
Added Tensorflow inspection example to the WF notebook

### PR DESCRIPTION
**Pull request by CermakM**
Depens-On: thoth-station/amun-api#265

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   notebooks/workflows/amun-inspection-workflow.ipynb

### CODE REVIEW

**Comment by CermakM** [comment]
Depens-On: https://github.com/thoth-station/amun-api/pull/265

**Comment by CermakM** [comment]
[(3) Change Amun calls to workload-operator to create workflow instance](https://trello.com/c/hQFLuQXu/836-3-change-amun-calls-to-workload-operator-to-create-workflow-instance)

**Review by goern — APPROVED** [review]

File: notebooks/workflows/amun-inspection-workflow.ipynb [hunk 1]
```diff
@@ -10,7 +10,7 @@
    },
```
Comment (pacospace) [inline notebooks/workflows/amun-inspection-workflow.ipynb:10]: <p>The inspection id should include the inspection identifer from specification :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (pacospace) [inline notebooks/workflows/amun-inspection-workflow.ipynb:10]: <p>Same comment as above regarding the inspection id</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Pull request merged**